components/autofill/src/db/schema.rs

```diff
@@ -73,120 +75,132 @@ const CREATE_SHARED_SCHEMA_SQL: &str = include_str!("../../sql/create_shared_sch
 const CREATE_SHARED_TRIGGERS_SQL: &str = include_str!("../../sql/create_shared_triggers.sql");
 const CREATE_SYNC_TEMP_TABLES_SQL: &str = include_str!("../../sql/create_sync_temp_tables.sql");
 
-// The schema version - changes to this typically require a custom
-// migration code.
-const VERSION: i64 = 2;
+pub(super) fn migration_logic() -> MigrationLogic {
+    MigrationLogic {
+        name: "autofill db".to_string(),
+        start_version: 0,
```

Comment:
The docs say that `start_version` is "usually 1", so it may be worth a comment here about why it's zero.